src/components/PostControls/PostMenu/PostMenuItems.tsx

```diff
@@ -470,6 +470,7 @@ let PostMenuItems = ({
           <>
             <Menu.Divider />
             <Menu.Group>
+              <Menu.LabelText>{_(msg`Tell feed provider:`)}</Menu.LabelText>
```

Comment:
```suggestion
              <Menu.LabelText>
                {_(
                  msg({
                    message: `Tell feed provider:`,
                    comment:
                      `Visible label for group of show more/show less options in post context menu in certain feeds which enable feedback`,
                  }),
                )}
              </Menu.LabelText>
```
A comment for translators might be helpful here? Although, maybe it's not too necessary as the string will be visible in the UI 🤔